### PR TITLE
PW-21 Implement cypress E2E tests

### PR DESCRIPTION
**Pull request by MCFreddie777**

### CODE REVIEW

**Review by OliverKanat — COMMENTED** [review]
prilis vela kodu

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "serve": "node scripts/start-web-server.js --port 8080",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
+    "lint": "vue-cli-service lint 'src/**' 'cypress/**'",
```
Comment (Hemii) [inline package.json]: Čo sa tu stane ?

File: src/views/Home.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="bg-primary">
+	<div class="bg-primary" data-e2e-id="homePage">
```
Comment (Hemii) [inline src/views/Home.vue]: Tiež by to chcelo rozdeliť ako nižšie, viac argumentov - viac riadkov

File: src/views/Documents.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="bg-gray-200">
+	<div class="bg-gray-200" data-e2e-id="documentsPage">
```
Comment (Hemii) [inline src/views/Documents.vue]: do 2 riadkov argumenty ?

File: src/views/Documents.vue [hunk 4]
```diff
@@ -62,15 +67,15 @@ export default {
...
 				],
-				'Retrospektívy': [
+				Retrospektívy: [
 					{
 						name: 'sprint_0_retro.pdf',
 						date: '22.10.2020',
 						link: '/download/sprint_0_retro.pdf'
 					}
 				],
 				'Exporty úloh': [],
```
Comment (Hemii) [inline src/views/Documents.vue]: Netreba tiež odstrániť úvodzovky ?

File: docs/E2E_TESTS.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# E2E Tests
+
+This documentation covers everything about developing and running E2E tests in Exponea.
```
Comment (Hemii) [inline docs/E2E_TESTS.md]: Exponea ? má to tam byť ?

**Review by Hemii — DISMISSED** [review]
Reviewed ✔️ 
Left some comments

File: package.json (hunk 1 above)
Comment (MCFreddie777) [reply package.json]: I just specified the correct paths to linter - so it'll catch all files in the `src`  and the `cypress` and it'll lint it (check the syntax against the pre-set rules).

**Review by OliverKanat — DISMISSED** [review]

**Review by Hemii — APPROVED** [review]
Approved ✔️ 
*Noice*

**Pull request merged**